cmd/loop/loopin.go

```diff
@@ -68,6 +74,7 @@ var (
 			labelFlag,
 			verboseFlag,
 			routeHintsFlag,
+			privateFlag,
```

Comment:
General style nit here, **not for this PR** but just generally in loop dev. 

I find that it makes sense to start with the changes _within_ loop, then fan them out. Eg in this case: 
- Add a `Private` bool to [LoopInRequest](https://github.com/lightninglabs/loop/blob/4c58d0c5e8eac5506e083caa1d84ad79d9b407bf/interface.go#L171) and add `SelectHopHints` if `Private` is true in the `Client`'s [LoopIn](https://github.com/lightninglabs/loop/blob/master/client.go#L508) method
- At this point, this logic is never triggered, since the `Private` bool is always false
- Next, update `client.proto` to add a `Private` bool to the [LoopInRequest](https://github.com/lightninglabs/loop/blob/master/looprpc/client.proto#L212) message and update `swapClientServer`'s [LoopIn](https://github.com/lightninglabs/loop/blob/4c58d0c5e8eac5506e083caa1d84ad79d9b407bf/loopd/swapclient_server.go#L598) method to set the value on the [LoopInRequest](https://github.com/lightninglabs/loop/blob/4c58d0c5e8eac5506e083caa1d84ad79d9b407bf/loopd/swapclient_server.go#L623) that you pass to the client
- At this point, the `loopd` API now supports the `Private` bool, although it is never set on the cli
- Finally, we surface this value on the cli, by adding a flag and setting it on the [LoopInRequest](https://github.com/lightninglabs/loop/blob/master/cmd/loop/loopin.go#L164) that the cli passes to `loopd`

This "fan out" means that once the flag is added on the cli, it's already implemented in the loop client. If you start with the cli, then checking out this commit actually has a meaningless flag, since the functionality hasn't been included yet.